plugins/simpleadminhacks-paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/BuildLimitsConfig.java

```diff
@@ -0,0 +1,49 @@
+package com.programmerdan.minecraft.simpleadminhacks.configs;
+
+import com.programmerdan.minecraft.simpleadminhacks.SimpleAdminHacks;
+import com.programmerdan.minecraft.simpleadminhacks.framework.SimpleHackConfig;
+import com.programmerdan.minecraft.simpleadminhacks.framework.utilities.BuildLimit;
+import org.bukkit.configuration.ConfigurationSection;
+
+import java.util.List;
+
+public final class BuildLimitsConfig extends SimpleHackConfig {
+	private boolean enabled;
+	private BuildLimit[] buildLimits;
+
+	public BuildLimitsConfig(SimpleAdminHacks plugin, ConfigurationSection base) {
+		super(plugin, base);
+	}
+	public BuildLimitsConfig(ConfigurationSection base) {
+		super(SimpleAdminHacks.instance(), base);
+	}
+
+	@Override
+	protected void wireup(ConfigurationSection config) {
```

Comment:
Put this all in onEnable and then you don't have to worry about checking if the hack is enabling in the config. Also, don't bother logging that the hack is enabled, and let errors throw, since that will disable the hack (or rethrow with a nicer message if you want)